pkg/image/api/helper.go

```diff
@@ -9,8 +9,13 @@ import (
 	"github.com/golang/glog"
 )
 
-// DockerDefaultNamespace is the value for namespace when a single segment name is provided.
-const DockerDefaultNamespace = "library"
+const (
+	// DockerDefaultNamespace is the value for namespace when a single segment name is provided.
+	DockerDefaultNamespace = "library"
```

Comment:
@deads2k @ncdc I can actually scratch this change as it is unrelated to this PR.